espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/InteractionHelper.kt

```diff
@@ -9,6 +12,12 @@ object InteractionHelper {
     @Synchronized
     fun getUiDevice(): UiDevice {
         if (uiDevice == null) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+                // The flag is necessary not to stop running accessibility service
+                // https://github.com/appium/appium/issues/4910
+                // https://developer.android.com/reference/android/app/UiAutomation.html#FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES
+                Configurator.getInstance().uiAutomationFlags = UiAutomation.FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES
```

Comment:
we probably should do 

```
                Configurator.getInstance().uiAutomationFlags = Configurator.getInstance().uiAutomationFlags | UiAutomation.FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES;
```

In order to not corrupt the current flags

Reply:
I would also prefer to have this option configurable to make sure it does not affect current tests